### PR TITLE
add edit form and avatar to user

### PR DESCRIPTION
**Pull request by SBlazhko**
screenshot from 2017-02-08 08-10-05
screenshot from 2017-02-08 08-10-20
screenshot from 2017-02-08 08-10-31

### CODE REVIEW

File: app/views/shared/_header.html.slim [hunk 1]
```diff
@@ -20,15 +20,16 @@ header.main-header
...
               span.hidden-xs = fa_icon 'angle-down', right: true, text: current_user.name
               
             ul.dropdown-menu
               li.user-header
-                / avatar
+                = image_tag(current_user.avatar.thumb, class: 'img-circle')
                 
                 p
-                  / #{current_user.name} - #{current_user.role}
+                  | #{current_user.name} - #{current_user.role}
```
Comment (IlyaUmanets) [inline app/views/shared/_header.html.slim]: We should use helper.

File: app/views/shared/_header.html.slim [hunk 2]
```diff
@@ -20,15 +20,16 @@ header.main-header
...
                   small = t('.signed_up') 
                   small #{current_user.created_at.strftime('%d %b. %Y')}
```
Comment (IlyaUmanets) [inline app/views/shared/_header.html.slim]: Same here

File: app/views/shared/_header.html.slim (hunk 1 above)
Comment (as-sergey-b) [inline app/views/shared/_header.html.slim]: fix

File: app/views/shared/_header.html.slim (hunk 2 above)
Comment (as-sergey-b) [inline app/views/shared/_header.html.slim]: fixed

File: app/uploaders/avatar_uploader.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # def default_url(*args)
```
Comment (IlyaUmanets) [inline app/uploaders/avatar_uploader.rb]: remove this method please

Comment (as-sergey-b) [inline app/uploaders/avatar_uploader.rb]: fixed

**Pull request merged**